packages/client/hmi-client/src/components/workflow/tera-simulation-workflow.vue

```diff
@@ -88,6 +90,13 @@ const testOperation: Operation = {
 	isRunnable: true
 };
 
+function appendOutputPort(index: number, type: string) {
+	nodes.value[index].outputs.push({
+		id: nodes.value[index].outputs.length.toString(),
+		type
+	});
+}
+
```

Comment:
Still not quite right. 

1. "node.id" is not the same as integer-index, node.id is actually defined as a string
2. Here we would actually want the value produced (model-config) in the value field 

So in short, your emit should pass back and store at least the model-config output you get from "run".

```
export interface WorkflowPort {
	id: string;
	type: string;
	label?: string;
	value?: any;
}```